cli/azd/pkg/infra/azure_resource_types.go

```diff
@@ -72,6 +74,10 @@ func GetResourceTypeDisplayName(resourceType AzureResourceType) string {
 		return "Azure Database for PostgreSQL flexible server"
 	case AzureResourceTypeCDNProfile:
 		return "Azure Front Door / CDN profile"
+	case AzureResourceTypeLoadTest:
+		return "Load Tests"
+	case AzureResourceTypeVirtualNetwork:
+		return "Virtual Network"
 	}
 
 	return ""
```

Comment:
Instead of not displaying the resource if it doesn't have a matching case statement, can't we output the resource name?

Reply:
I think it could lead the user to ask why resources show up differently. Also, for resources that follow abbreviation guidelines, something like `app-api-2m7uj3nhjpbni` alone seems hard to understand.

The real solution is to consume translation assets from the Azure Portal. We're still talking to folks from Portal to make this a reality.